Description: fix detection of application's directory scheme
 The application tries to detect if it is installed in /usr/bin -
 then it will use a FHS compliant directory scheme. The .startsWith()
 call is wrong, as .applicationDirPath() doesn't return a trailing
 slash. This patch was accepted upstream with
 <URL: https://github.com/samaaron/sonic-pi/pull/1161>.
Forwarded: yes
Author: Hanno Zulla <kontakt@hanno.de>
Last-Update: 2016-04-18

Index: sonic-pi/app/gui/qt/mainwindow.cpp
===================================================================
--- sonic-pi.orig/app/gui/qt/mainwindow.cpp	2016-04-15 16:30:04.756592919 +0200
+++ sonic-pi/app/gui/qt/mainwindow.cpp	2016-04-15 16:31:00.355804883 +0200
@@ -102,7 +102,7 @@
 MainWindow::MainWindow(QApplication &app, bool i18n, QSplashScreen* splash)
 #endif
 {
-  if (QCoreApplication::applicationDirPath().startsWith("/usr/bin/")) {
+  if (QCoreApplication::applicationDirPath().startsWith("/usr/bin")) {
 
     // use FHS directory scheme:
     // Sonic Pi is installed in /usr/bin from a Linux distribution's package
@@ -111,7 +111,7 @@
     ruby_server_path = "/usr/lib/sonic-pi/server/bin/sonic-pi-server.rb";
     sample_path = "/usr/share/sonic-pi/samples";
 
-  } else if (QCoreApplication::applicationDirPath().startsWith("/opt/")) {
+  } else if (QCoreApplication::applicationDirPath().startsWith("/opt")) {
 
     // use /opt directory scheme:
     // Sonic Pi is installed in /opt from the Raspbian .deb package
